SampleApp/SampleApp/Screens/CreatablesView.swift

```diff
@@ -16,6 +16,13 @@ struct CreatablesView: View {
 
     var body: some View {
         ScrollView {
+            HStack {
+                TextField("location", text: $viewModel.location)
+                    .roundedBorderTextField()
+                Button("CALL") {
```

Comment:
Should it be "CALL"? Is that a string from a design?

Reply:
@rossrossp There's no design for Sample app, on Android the string is "DO CA CALL".
I can change it to be the same as Android, or do you think we should ask design team for this?